### ISSUE TITLE
Lots o' Errorz

### ISSUE THREAD

**Issue by the-dansan** [SonarSource/sonar-loc-count#30]
Hi there, Sonar sent us this tool in order to help us evaluate the product as a potential customer. However I'm so far unable to get it to work in ubuntu. Would love some assistance with what I might be missing.

![image](https://github.com/SonarSource/sonar-loc-count/assets/18700228/eece348d-e61a-460d-88e0-1359b27d3029)

**Comment by sylvain-combe-sonarsource** [comment]
Hello @the-dansan 
please address the issue according to the script input [here](https://github.com/SonarSource/sonar-loc-count/blob/4bca418b43d97842dcc52fb7c6217343d6f9fca8/bitbucket_org.sh#L37). On Linux sed **is** gsed
And let us know if the script goes further.

Thanks

**Comment by the-dansan** [comment]
Thank you. Are you suggesting I alter the script to remove that alias?

**Comment by sylvain-combe-sonarsource** [comment]
Yes, you're supposed to alter the script on Linux systems as follows:
`SED="sed" `

**Comment by sylvain-combe-sonarsource** [comment]
And we'll probably improve this with https://github.com/SonarSource/sonar-loc-count/pull/31

**Comment by the-dansan** [comment]
Yes, definitely running now. It looks like we have lots of repos with hundreds of leftover branches. Do you have a handy way to forward an include list for branch names to cloc?

**Comment by sylvain-combe-sonarsource** [comment]
This is planned with #25 
As of today you may
* slightly update the script so that it ignores parameter n4 (removing  lines 48 and 55 to 58 should do)
* add the name of the main branch for all repos as parameter n5
And let me know if this works

**Comment by the-dansan** [comment]
Looks like I can make that work. Thank you!
